### ISSUE TITLE
Text contrast

### ISSUE THREAD

**Issue by anormand** [unimelb/The-University-of-Melbourne-Web-Templates#13]
### Problem:

The text ‘A pioneering research university, The University of Melbourne has a the largest cohort of research students in Australia’ has a contrast ratio with the background of 1.2:1 at worst.
### WCAG Guideline

Success Criteria 1.4.3 - Contrast

The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except incidental text or images and logotypes. Large text (over 18pt or 14 pt bold) has a contrast ratio of at least 3:1.
### Remedy:

Ensure that the contrast ratio between text and background is at least 4.5:1

**Comment by ptagell** [comment]
Note: Andrew tests bottom left of the word.

**Comment by anormand** [comment]
WCAG suggests the following solution (http://www.w3.org/TR/2010/NOTE-WCAG20-TECHS-20101014/G18):

If the background or the letters vary in relative luminance (or are patterned) then the background around the letters can be chosen or shaded so that the letters maintain a 4.5:1 contrast ratio with the background behind them even if they do not have that contrast ratio with the entire background.

For example, if a letter is lighter at the top than it is a the bottom, it may be difficult to maintain the contrast ratio between the letter and the background over the full letter. In this case, the designer might darken the background behind the letter, or add a thin black outline (at least one pixel wide) around the letter in order to keep the contrast ratio between the letter and the background above 4.5:1.
